addons/hr_expense/wizard/hr_expense_submit_lines.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from openerp import api, fields, models
+
+
+class HrExpenseSubmitLines(models.TransientModel):
+
+    _name = "hr.expense.submit.lines.wizard"
+    _description = "Hr Expense Submit Lines wizard"
+
+    sheet_name = fields.Char(string='Sheet Name', required=True)
+
+    @api.multi
+    def expense_submit(self):
+        self.ensure_one()
+        sheet_id = self.env['hr.expense.sheet'].create({'name': self.sheet_name})
+        context = dict(self._context or {})
+        active_ids = context.get('active_ids', [])
```

Comment:
Directly check in self.env.context, context does not seem to be used (probably an old-API construct).